platform/windows/windows_terminal_logger.cpp

```diff
@@ -40,7 +40,7 @@ void WindowsTerminalLogger::logv(const char *p_format, va_list p_list, bool p_er
 		return;
 	}
 
-	const unsigned int BUFFER_SIZE = 16384;
+	const unsigned int BUFFER_SIZE = 163840;
```

Comment:
Is this necessary?

Reply:
I needed this in order to debug some issues, since error messages on Windows were being truncated.

I will revert though for the purpose of this PR.